cradle-cassandra/src/test/java/com/exactpro/cradle/cassandra/integration/messages/GroupedMessageIteratorProviderTest.java

```diff
@@ -51,10 +57,9 @@ public class GroupedMessageIteratorProviderTest extends BaseCradleCassandraTest
 
     private final long storeActionRejectionThreshold = new CoreStorageSettings().calculateStoreActionRejectionThreshold();
 
-    private List<GroupedMessageBatchToStore> data;
     private List<StoredGroupedMessageBatch> storedData;
     private CassandraOperators operators;
-    private ExecutorService composingService = Executors.newSingleThreadExecutor();
+    private final ExecutorService composingService = Executors.newFixedThreadPool(3);
```

Comment:
I added more than 1 thread because Cradle Storage use thread pool with size 5 by default.
Maybe these or future tests in this class show a concurrent problem